scalafmt-tests/src/test/resources/scala3/Enum.stat

```diff
@@ -129,9 +129,8 @@ enum Foo(val x: Int) {
 }
 >>>
 enum Foo(val x: Int) {
-  case Bar
-      extends Foo(1) with X
-      with Y with Z
+  case Bar extends Foo(1)
```

Comment:
This seems wrong now. With `binPack.parentConstructors = Always` wasn't the previous state correct?